.gitignore

```diff
@@ -0,0 +1,9 @@
+.directory
+__pycache__
+pull_request_Information.md
```

Comment:
That's actually why I wasn't tracking `.gitignore` before (because mine currently lists occasional scratchpad files I've been using for development).

Do you have a recommended approach for this workflow?

Reply:
Yes you should track a .gitignore with just the lines added that are relevant for everyone. Note that you can stage only certain lines of a file in git using `git add --patch`.

It's also fine for you to have local files that aren't ignored. I rarely do `git add --all`. Instead I just add the specific files that should be added (or all updated ones `git add --update`).

Reply:
I've implemented reading the TSV file from the URL you provided in e135948.

Reply:
I've pared down the `.gitignore` file per your specifications in 2b8e44.